fds/utils.py

```diff
@@ -52,6 +52,8 @@ def execute_command(command: Union[str, List[str]], shell: bool = False, capture
         return subprocess.CompletedProcess(command, output.returncode, b''.join(stdout), b''.join(stderr))
     else:
         output = subprocess.run(command, shell=shell)
+        if output.returncode not in ignorable_return_codes:
+            raise Exception()
```

Comment:
I think it would be best to show the user what is the command that failed, its output and status codes.
Otherwise we'll get users reporting bugs in FDS when it's actually something simple like "could not find git executable" or something like that

Reply:
Yes sure makes sense